package.json

```diff
@@ -22,81 +22,83 @@
-    "@testing-library/dom": "8.12.0",
-    "@testing-library/jest-dom": "5.16.3",
-    "@testing-library/react": "12.1.4",
+    "@testing-library/dom": "8.13.0",
+    "@testing-library/jest-dom": "5.16.4",
+    "@testing-library/react": "12.1.5",
     "@testing-library/react-hooks": "7.0.2",
     "@testing-library/user-event": "13.5.0",
-    "@typescript-eslint/eslint-plugin": "5.18.0",
-    "@typescript-eslint/parser": "5.18.0",
+    "@typescript-eslint/eslint-plugin": "5.22.0",
+    "@typescript-eslint/parser": "5.22.0",
     "@wojtekmaj/enzyme-adapter-react-17": "0.6.7",
     "babel-jest": "27.5.1",
-    "babel-loader": "8.2.4",
+    "babel-loader": "8.2.5",
     "babel-plugin-module-resolver": "4.1.0",
     "babel-plugin-typescript-to-proptypes": "2.0.0",
     "concurrently": "7.1.0",
     "css-loader": "5.2.7",
-    "cypress": "9.5.3",
+    "cypress": "9.6.0",
     "deepmerge": "4.2.2",
     "enzyme": "3.11.0",
     "enzyme-adapter-react-16": "1.15.6",
     "enzyme-to-json": "3.6.2",
-    "eslint": "8.12.0",
+    "eslint": "8.14.0",
     "eslint-config-prettier": "8.5.0",
-    "eslint-config-react-app": "7.0.0",
+    "eslint-config-react-app": "7.0.1",
     "eslint-plugin-cypress": "2.12.1",
     "eslint-plugin-flowtype": "8.0.3",
-    "eslint-plugin-import": "2.25.4",
+    "eslint-plugin-import": "2.26.0",
     "eslint-plugin-jsx-a11y": "6.5.1",
     "eslint-plugin-prettier": "4.0.0",
     "eslint-plugin-react": "7.29.4",
-    "eslint-plugin-react-hooks": "4.4.0",
-    "eslint-plugin-testing-library": "5.2.1",
+    "eslint-plugin-react-hooks": "4.5.0",
+    "eslint-plugin-testing-library": "5.3.1",
     "jest": "27.5.1",
     "npm-package-json-lint": "5.4.2",
     "prettier": "2.6.2",
     "react": "17.0.2",
     "react-docgen-typescript-loader": "3.7.2",
     "react-dom": "17.0.2",
-    "sass": "1.49.11",
+    "sass": "1.51.0",
     "sass-loader": "10.2.1",
     "style-loader": "2.0.0",
-    "stylelint": "14.6.1",
+    "stylelint": "14.8.2",
     "stylelint-config-prettier": "9.0.3",
     "stylelint-config-recommended-scss": "5.0.2",
     "stylelint-order": "5.0.0",
     "stylelint-prettier": "2.0.0",
     "ts-jest": "27.1.4",
-    "tsc-alias": "1.6.5",
-    "typescript": "4.6.3",
+    "tsc-alias": "1.6.7",
+    "typescript": "4.6.4",
     "vanilla-framework": "3.3.0",
     "wait-on": "5.3.0"
   },
   "dependencies": {
-    "@types/jest": "27.4.1",
-    "@types/node": "16.11.26",
-    "@types/react": "17.0.43",
-    "@types/react-dom": "17.0.14",
-    "@types/react-table": "7.7.10",
+    "@types/jest": "27.5.0",
+    "@types/node": "16.11.33",
+    "@types/react": "17.0.44",
+    "@types/react-dom": "17.0.16",
+    "@types/react-table": "7.7.11",
     "classnames": "2.3.1",
-    "nanoid": "3.3.2",
+    "nanoid": "3.3.4",
     "prop-types": "15.8.1",
     "react-table": "7.7.0",
     "react-useportal": "1.0.16"
   },
   "resolutions": {
+    "@types/react": "^17.0.44",
+    "@types/react-dom": "^17.0.16",
```

Comment:
Need to lock these types, because some packages start loading latest types for react v18 which don't work here.